cli/sawtooth_cli/batch_status.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+from sys import maxsize
+
+from sawtooth_cli.rest_client import RestClient
+from sawtooth_cli.parent_parsers import base_http_parser
+
+
+def add_batch_status_parser(subparsers, parent_parser):
+    """Adds arguments parsers for the batch_status commands
+
+        Args:
+            subparsers: Add parsers to this subparser object
+            parent_parser: The parent argparse.ArgumentParser object
+    """
+    epilog = '''details:
+        Fetches the committed statuses for a single batch.
+    '''
+    parser = subparsers.add_parser(
+        'batch_status', epilog=epilog,
```

Comment:
Change this to `batch-status`.  Operating under a no-underscores-in-command-args policy.